releasenotes/notes/quantize-digits-in-sql-ids-f846ad5f8938889d.yaml

```diff
@@ -0,0 +1,12 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    APM: Updated the obfuscator to quantize digits in IDs of SQL statement in addition to table names,
```

Comment:
Updated, although I used "replace digits" rather than "obfuscate" to keep it consistent with the term used everywhere else. https://github.com/DataDog/datadog-agent/pull/8860/commits/bec8b338f1048c08f711e9197a2568ef723c7138